h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostTest.java

```diff
@@ -583,6 +674,55 @@ public void MNIST() {
     }
   }
 
+  @Test
+  public void MNIST_LightGBM() {
+    Frame tfr = null;
+    Frame preds = null;
+    XGBoostModel model = null;
+    Scope.enter();
+    try {
+      // Parse frame into H2O
+      tfr = parse_test_file("bigdata/laptop/mnist/train.csv.gz");
```

Comment:
Some of the tests that use "bigdata" are safe-guarded and don't fail when the data is not available. Some don't. I think this is fine :)